run_dir/design/bioinfo_tab.html

```diff
@@ -42,9 +42,7 @@
     {% include "bioinfo_tab/run_lane_sample_view.html" %}
     {% include "bioinfo_tab/sample_run_lane_view.html" %}
 
-    <button disabled class="btn btn-default" id="bioinfo-show-history">Show Edit History</button>
-    <pre id="bioinfo-history-dump" style="display:none; margin-top:20px;"></pre>
-
+    <button id="bioinfo-show-history-button" class="btn btn-default" data-target="#bioinfo-show-history">Show Edit History</button>
     <script src="/static/js/bootstrap-datepicker.min.js"></script>
```

Comment:
Would be good to move these to the end of the file, might get lost in the middle here.